### PR TITLE
Exercises backup guyer

### PR DESCRIPTION
**Pull request by guyer**

### CODE REVIEW

File: guyer-jonathan/backup/compare_manifests.py [hunk 1]
```diff
@@ -0,0 +1,198 @@
+import csv
+
+def read_manifest(manifest_file=None):
+    """
+    Read a manifest `.csv` file into a dictionary
```
Comment (gvwilson) [inline guyer-jonathan/backup/compare_manifests.py:5]: docstrings for the win!

File: guyer-jonathan/backup/compare_manifests.py [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+        deleted : dict
+            Files are in the first hash but neither their names nor their hashes are in the second
+            (file hash keys with file path as value)
+        added : dict
+            Files are in the second hash but neither their names nor their hashes are in the first
+            (file hash keys with file path as value)
+    """
+    manifestA = read_manifest(fileA)
+    manifestB = read_manifest(fileB)
+    reversedA = {}
```
Comment (gvwilson) [inline guyer-jonathan/backup/compare_manifests.py:56]: why "reversed"? (I understand what the code is doing, but why the name?)

File: guyer-jonathan/backup/compare_manifests.py [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+        print()
+
+def test_read():
+    manifest = read_manifest("test/123456789.csv")
+
+    expected = {
+        'deadbeef': 'path/to/t-bone.txt',
+        'cafeb0ba': 'path/to/boba/are_gross.txt',
+        'decafbad': 'path/to/caffeine.py',
+        '8badf00d': 'path/to/do/not/feel/good.py',
```
Comment (gvwilson) [inline guyer-jonathan/backup/compare_manifests.py:136]: hadn't seen this one before :-)

File: guyer-jonathan/backup/compare_manifests.py (hunk 2 above)
Comment (guyer) [reply guyer-jonathan/backup/compare_manifests.py:56]: I was using those for a "reverse lookup" of filename to filehash.

**Pull request merged**